src/gordon_gcp/plugins/service/gdns_publisher.py

```diff
@@ -179,16 +179,9 @@ def __init__(self, config, success_channel, error_channel, http_client):
         self.error_channel = error_channel
         self.http_client = http_client
         self.publish_wait_timeout = self.config.get('publish_wait_timeout', 60)
-        self.managed_zone = self.config['managed_zone']
-        self.dns_zone = self.config['dns_zone']
         self.project = self.config['project']
         self.default_ttl = self.config['default_ttl']
         self.api_version = self.config.get('api_version', 'v1')
-        self.base_changes_url = BASE_CHANGES_ENDPOINT.format(
-            version=self.api_version, project=self.project,
-            managedZone=self.managed_zone)
-        self.resource_records_url = RESOURCE_RECORDS_ENDPOINT.format(
-            project=self.project, managedZone=self.managed_zone)
```

Comment:
Maybe these constants should `GDNSPublisher` class constants rather than module constants, so they can be inherited as well.

Reply:
Good idea.